patches/minecraft/net/minecraft/entity/passive/BeeEntity.java.patch

```diff
@@ -0,0 +1,53 @@
+--- a/net/minecraft/entity/passive/BeeEntity.java
++++ a/net/minecraft/entity/passive/BeeEntity.java
+@@ -68,7 +68,7 @@ import net.minecraft.state.properties.DoubleBlockHalf;
+ import net.minecraft.tags.BlockTags;
+ import net.minecraft.tags.ItemTags;
+ import net.minecraft.tags.Tag;
+-import net.minecraft.tileentity.BeehiveTileEntity;
++import net.minecraft.tileentity.IBeehiveTileEntity;
```

Comment:
Do not change imports in patches. Use fully qualified names for new class references, and leave now unused imports alone.